platforms/documentation/docs/src/docs/release/notes.md

```diff
@@ -84,6 +84,33 @@ gradle init --use-defaults --type kotlin-library --no-comments
 
 It is possible to persist the preference by setting the `org.gradle.buildinit.comments` property to `false`.
 
+<a name="replace-method"></a>
+#### Updating lazy property based on its current value with `replace()`
+
+[Lazy configuration](userguide/lazy_configuration.html) delays calculating a property’s value until it is required for the build.
+Sometimes it is necessary to modify the property based on its current value, for example, by appending something to it.
+Previously, the only way to do that was to obtain the current value explicitly by calling `Property.get()`:
+```
+val property = objects.property<String>()
+property.set("some value")
+property.set("${property.get()} and more" })
+
+println(property.get()) // "some value and more""
+```
+This could lead to performance issues like configuration cache misses.
+Trying to build the value in a lazy manner, for example, by using `property.set(property.map { "$it and more" })`, causes build failure because of a circular reference evaluation.
```

Comment:
```suggestion
Trying to build the value lazily, for example, by using `property.set(property.map { "$it and more" })`, causes build failure because of a circular reference evaluation.
```